lib/tasks/export_dags.rake

```diff
@@ -1,10 +1,16 @@
-task :export_dags, [:sample_id] => :environment do |_, args|
+task :export_dags, [:sample_id, :print, :step] => :environment do |_, args|
```

Comment:
I approved the PR based on the jbuilder files. I defer to @kislyuk about whether the changes to the export_dags.rake file are fine -- a priori I'd probably err on the side of only changing the dag jbuilder for this PR